autoload/go/lsp/message.vim

```diff
@@ -322,6 +324,14 @@ function! go#lsp#message#ConfigurationResult(items) abort
       let l:config.local = l:local
     endif
 
+    if l:gofumpt isnot v:null
+      let l:config.gofumpt = l:local
```

Comment:
Isn't this a typo? I suppose it should be `= l:gofumpt`.

Reply:
Yeah, that's a typo. PR coming up.